jetbrains-core/tst/software/aws/toolkits/jetbrains/core/MockClientManager.kt

```diff
@@ -64,7 +63,6 @@ class MockClientManager : AwsClientManager() {
          */
         fun useRealImplementations(disposable: Disposable) {
             val clientManager = AwsClientManager()
-            Disposer.register(disposable, clientManager)
```

Comment:
do these get disposed automatically?

Reply:
Yes, I dove into the code and confirmed it.

https://github.com/JetBrains/intellij-community/blob/210998614a279c714bdc1652604bad5bfb3237e4/platform/service-container/src/com/intellij/serviceContainer/BaseComponentAdapter.kt#L158